Sprite.java

```diff
@@ -0,0 +1,122 @@
+import org.json.simple.JSONObject;
+
+/**
+ * Sprite.java
+ *
+ * @author Michelle Melton
+ * @author Kara Beason
+ * @author Cydney Caldwell
+ */
+
+/**
+ * Class for a sprite within a Scratch submission.
+ *
+ * @author Michelle Melton
+ * @author Kara Beason
+ * @author Cydney Caldwell
+ * @version Apr 2017
```

Comment:
That's how all the files are formatted. I think I picked this up from one of his prior examples, because I remember thinking it was repetitive as well. Do you know what the standard for author notation is?